src/interceptors/WebSocket/WebSocketServerConnection.ts

```diff
@@ -117,6 +117,14 @@ export class WebSocketServerConnection {
       this.transport.onIncoming(event)
     })
 
+    // Forward server errors to the WebSocket client as-is.
+    // We may consider exposing them to the interceptor in the future.
+    ws.addEventListener('error', () => {
```

Comment:
We may consider exposing this on the `server` event target for the consumer in the future. WebSocket errors aren't that useful to listen to so, for now, we simply forward.